deis.go

```diff
@@ -89,7 +89,7 @@ type Client struct {
 // controller is unsafe.
 //
 // If the SDK detects an API version mismatch, it will return ErrAPIMismatch.
-const APIVersion = "2.1"
+const APIVersion = "2.2"
```

Comment:
does this patch need to be delayed until the 2.2 release? also, we can toggle this at build time